lib/job_queue/ext_job.cpp

```diff
@@ -1107,6 +1113,21 @@ const stringlist_type * ext_job_get_arglist( const ext_job_type * ext_job ) {
     return ext_job->argv;
 }
 
+const stringlist_type * ext_job_get_argvalues( const ext_job_type * ext_job ) {
```

Comment:
would it not make sense to create this when setting the arguments and not every time when extracting?

Reply:
It would require the `set_private_args` to be called before the `set_defined_args`, so I'm not sure.. We could do some checking if the `arg_values` have been resolved, it would add more state to the structure though.

Reply:
Is there some configuration where set_private_args and set_defined_args are called on their own?

Reply:
We do explicitly in the `forward_model_parse_job_deprecated_args` and `forward_model_parse_job_args`